app/lib/server/functions/notifications/mobile.js

```diff
@@ -15,7 +15,7 @@ Meteor.startup(() => {
 
 export async function getBadgeCount(userId) {
 	const [result = {}] = await SubscriptionRaw.aggregate([
-		{ $match: { 'u._id': userId } },
+		{ $match: { 'u._id': userId, archived: false } },
```

Comment:
This isn't the only place in the codebase that counts unreads.  See my commit message in RC commit d885e41de77e87cc59c97edfef7699252daa11da.  We will need either to update the _other_ places that count unreads or consider extracting out the query into a central method.

Reply:
**UPDATE:** We didn't add the DB index mentioned above.  I don't think the index will be so expensive, since it will be sparse, but on the other hand, there will probably not be so many room subscriptions for a given user.  Maybe we can leave out the index and add it if/when we see it being a bottleneck.